### PR TITLE
Fix description of automatic in ICT

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

**Comment by mfojtik** [comment]
@soltysh PTAL

File: pkg/deploy/api/v1/types.go [hunk 1]
```diff
@@ -278,6 +278,8 @@ const (
 // DeploymentTriggerImageChangeParams represents the parameters to the ImageChange trigger.
 type DeploymentTriggerImageChangeParams struct {
 	// Automatic means that the detection of a new tag value should result in a new deployment.
+	// Note that if this is set to false, the image field in the container specification will
+	// not be automatically resolved to an image stream tag specified in field from.
 	Automatic bool `json:"automatic,omitempty" description:"whether detection of a new tag value should trigger a deployment"`
```
Comment (soltysh) [inline pkg/deploy/api/v1/types.go]: Add that comment to description as well, this way we'll have it in the swagger spec ppl are looking at.

**Comment by kargakis** [comment]
cc: @ironcladlou

**Comment by ironcladlou** [comment]
Consider adding a note in the deployment section of the user's guide?

File: pkg/deploy/api/v1/types.go (hunk 1 above)
Comment (smarterclayton) [inline pkg/deploy/api/v1/types.go]: Description tag is days away from being gone

**Comment by soltysh** [comment]
@mfojtik can you address @ironcladlou comment about extending our user guide and you're good to go.

**Comment by mfojtik** [comment]
@soltysh i'm getting there

**Comment by kargakis** [comment]
Superseded by https://github.com/openshift/origin/pull/8746/commits/e3510a74fd5c964c3069886240fa9edfd5713c8f

**Pull request closed without merging**